src/utils/get-comment-registry.ts

```diff
@@ -368,7 +369,12 @@ export function attachCommentsToOutputNodes(
     /** Original declaration, not the re-sorted output-node! */
     firstImport: ImportDeclaration,
 ) {
-    if (outputNodes.length === 0) {
+    if (!Array.isArray(commentEntriesFromRegistry)) {
```

Comment:
How would this happen without throwing a TypeScript compilation error?

Reply:
I concur that TypeScript would prevent that. [Responded further here: https://github.com/IanVS/prettier-plugin-sort-imports/pull/84#discussion_r1194435397]